src/Listener.php

```diff
@@ -64,10 +64,48 @@ public function setTagNameCase($case)
      *                                 It uses array_walk() internally.
      *                                 The suggested signature for the argument is:
      *                                 function (string &$value, string $tag);
+     *
+     * @deprecated
      */
     public function setTagValueProcessor(callable $processor = null)
```

Comment:
Could you trigger a `E_USER_DEPRECATED ` error and describe the `@deprecated` annotation like symfony does?

http://symfony.com/doc/current/contributing/code/conventions.html#deprecations